internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec.go

```diff
@@ -107,25 +113,25 @@ func verifyPipelines(spec *loggingv1.ClusterLogForwarderSpec, status *loggingv1.
 	inputs := sets.StringKeySet(spec.InputMap())
 
 	for i, pipeline := range spec.Pipelines {
-		// Don't allow empty names since this no longer mutates the spec
+		// Give anonymous pipelines internal "pipeline_[0..n]" names
```

Comment:
We don't allow anonymous pipelines and are correcting the documentation accordingling: https://issues.redhat.com/browse/LOG-4105